cpu/amd64/api.go

```diff
@@ -30,3 +30,107 @@ func MemoryAddWithCarry[
 
 // Return returns from the current function.
 func Return() Appender { return ret }
+
+// Aaa performs ASCII adjust after addition.
+//
+//asm:AAA
+func Aaa() Appender { return aaa }
```

Comment:
Remember to follow the naming conventions in this file, full words, instead of abbreviations, also please add the asm tags to the existing functions, `AddWithCarry` (ADC), `Add` (ADD), `MemoryAddWithCarry` (ADC)

You don't have to add any additional exports just yet, we can worry about that later.  For example, with Jump we will need to think about how to deal with offsets and `CALL` should support calling a register, not an offset directly. You can just remove the additional exports for now if that is easier (it probably is).